File: tuan02/bai37/bai37.jl
function dem(a::Vector{Int64})::Tuple{Int64, Int64, Int64}
    chan = 0
    le = 0
    am = 0
    for x in a
        if x < 0
            am += 1
        end
        if x % 2 == 0
            chan += 1
        else
            le += 1
        end
    end
    return chan, le, am
end

input_str = read(stdin, String)
tokens = split(input_str)
if !isempty(tokens)
    n = parse(Int64, tokens[1])
    a = [parse(Int64, tokens[i]) for i in 2:(n+1)]
    chan, le, am = dem(a)
    println("$chan $le $am")
end

File: tuan02/bai37/bai37.py
def dem(a: list[int]) -> tuple[int, int, int]:
    chan = 0
    le = 0
    am = 0
    for x in a:
        if x < 0:
            am += 1
        if x % 2 == 0:
            chan += 1
        else:
            le += 1
    return chan, le, am

if __name__ == "__main__":
    import sys
    input_data = sys.stdin.read().split()
    if input_data:
        n = int(input_data[0])
        a = [int(x) for x in input_data[1:n+1]]
        chan, le, am = dem(a)
        print(f"{chan} {le} {am}")